.github/workflows/pr_build.yml

```diff
@@ -48,7 +53,7 @@ jobs:
     strategy:
       max-parallel: 5
       matrix:
-        java: [8, 11, 15, 16, 17, 18, 19, 20 ]
+        java: [8, 11, 17, 15, 16, 18, 19, 20]
```

Comment:
@jeromevdl i'm sure you would tell me to sort these in the right order ;)

Reply:
actually I wanted to put the LTS first: 8, 11, 17, 21 and the others after, but finally removed 21